src/Test/Perf/Runner/Benchview.cs

```diff
@@ -1,22 +1,47 @@
-﻿using System;
+﻿using Roslyn.Test.Performance.Utilities;
+using System;
 using System.Collections.Generic;
-using System.Linq;
 using System.IO;
 using System.Xml;
 using static Roslyn.Test.Performance.Utilities.TestUtilities;
+using static Roslyn.Test.Performance.Runner.Tools;
 
-namespace Roslyn.Test.Performance.Utilities
+namespace Roslyn.Test.Performance.Runner
 {
-    internal class Tools
```

Comment:
Yep, there's a bunch of complaining.  

I'm going to have a documentation commit after the structure is approved.